.gitignore

```diff
@@ -5,3 +5,4 @@ target/
 public/dist
 .ensime*
 .DS_Store
+.bsp/
```

Comment:
Build Server Protocol (BSP) support was [added in sbt v1.4](https://www.scala-lang.org/blog/2020/10/27/bsp-in-sbt.html).